.github/workflows/test_pdm_env_macos.yaml

```diff
@@ -0,0 +1,56 @@
+---
+# configuration for GitHub Actions
+name: floras tests in PDM env - MacOS
+on:
+    push:
+    pull_request:
+
+jobs:
+    build:
+        name: Build and test
+        runs-on: ${{ matrix.os }}
+        strategy:
+            matrix:
+                os: ["macos-latest"]
+                python-version: [
+                    '3.10',
+                    '3.11',
+                    '3.12'
+                ]
+        steps:
+            - uses: actions/checkout@v4
+            - uses: pdm-project/setup-pdm@v4
+              with:
+                  python-version: ${{ matrix.python-version }}
+                  cache: true
+
+            - name: Install graphviz
+              run: |
+                 brew install graphviz
+                 pip install --no-cache-dir \
+                    --config-settings="--global-option=build_ext" \
+                    --config-settings="--global-option=-I$(brew --prefix graphviz)/include/" \
+                    --config-settings="--global-option=-L$(brew --prefix graphviz)/lib/" \
+                    pygraphviz
+
+            - name: Install Python dependencies
+              run: |
+                pdm install -v
```

Comment:
```suggestion
                export CFLAGS="-I $(brew --prefix graphviz)/include"
                export LDFLAGS="-L $(brew --prefix graphviz)/lib"
                pdm install -v
```

Reply:
Using environment variables as above works for me locally when installing via PDM on my Mac, and I tested the same on a temporary branch: https://github.com/slivingston/floras/actions/runs/11287876234/workflow